module/VuFind/sql/mysql.sql

```diff
@@ -409,3 +409,35 @@ CREATE TABLE `access_token` (
   CONSTRAINT `access_token_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
+
+CREATE TABLE `notifications_broadcasts` (
+                                            `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+                                            `broadcast_id` int(11) DEFAULT NULL,
+                                            `visibility` tinyint(1) DEFAULT NULL,
+                                            `visibility_global` tinyint(1) DEFAULT NULL,
+                                            `priority` int(11) DEFAULT NULL,
+                                            `author_id` int(11) DEFAULT NULL,
+                                            `content` text DEFAULT NULL,
+                                            `color` varchar(255) DEFAULT NULL,
+                                            `startdate` datetime DEFAULT NULL,
+                                            `endate` datetime DEFAULT NULL,
+                                            `change_date` datetime DEFAULT NULL,
+                                            `create_date` datetime DEFAULT NULL,
+                                            `language` varchar(255) DEFAULT NULL,
+                                            PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Comment:
The indentation of this is inconsistent with the rest of the file. Also, I'm not sure why the encoding is latin1 rather than utf8; it would probably be a good idea to adjust that for consistency.

Finally, we'll eventually need to make this work with PostgreSQL as well, but it's probably wise to finalize the design and functionality with MySQL first. I've added a TODO checkbox to the PR to keep track of this task.